install.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+if [ "$(id -u)" != "0" ]; then
+    echo "Run it as root"
+	echo
+	echo "usage: sudo bash install.sh --install | --uninstall"
+
+else
+	if [ "$1" = "--install" ] ; then
+		if [ ! -f /usr/bin/geckodriver ] && [ ! -f /usr/local/bin/geckodriver ] ; then
```

Comment:
there should be OR operator instead of AND like this
`if [ ! -f /usr/bin/geckodriver ] || [ ! -f /usr/local/bin/geckodriver ] ; then`